collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
+* In 1990, emissions in what is now the European Union amounted to 5 billion tonnes of CO<sub>2</sub>eq. In 2019, it was 3.7 billion tonnes. This statistic indicates that EU emissions have fallen by 24% over nearly thirty years. Total EU emissions have been falling more significantly since 2007, after the global financial crisis, in all sectors except transport.
+{% comment %}
+{% include preview-box.html
+    title="Where does the Czech Republic stand in terms of emissions?"
+    text="We have created a similar [infografiku](/infografiky/emise-cr-vyvoj) for the Czech Republic."
+    slug="emise-cr-vyvoj"
+%}
+{% endcomment %}
+
+## What we see in individual sectors
+
+* __Energy:__ Emissions from the energy sector fell by 39% compared to 1990 to 972 million tonnes of CO<sub>2</sub>eq per year. Emissions from the energy sector began to decline significantly after 2007. In recent years we have seen a more rapid decline, which can be expected to continue in the future, given the EU's commitment to achieving <glossary id="co2eq">climate neutrality</glossary> by 2050.
+Emissions in the energy sector come primarily from electricity and heat production. These emissions can be reduced through energy savings and the development of renewable and low-carbon energy sources. They also come from leakages from fuel extraction, fuel production and treatment and oil processing.
+* __Transport:__ Transport emissions increased by 33% compared to 1990 to 968 million tonnes of CO<sub>2</sub>eq per year. The detailed graph on the right shows a temporary drop in emissions after 2007 due to the global financial crisis and the subsequent economic recession. Since 2013, emissions can be seen to have risen again. GHG emissions from transport are primarily from the combustion of fossil fuels in the engines of road transport vehicles. (In 2019, 81% of all emissions from the transport sector were from transport, with aviation accounting for 15%.)
+It is possible to reduce transport emissions by switching to alternative fuels (e.g., biomethane, CNG, hydrogen or electricity with a simultaneous energy transformation), increasing the share of public transport (train and bus) and reducing the number of vehicles on the road. The volume of road transport can be reduced by increasing vehicle occupancy (carpooling) or generally decreasing the need for transportation (e.g. teleworking).
+* __Industrial processes:__ Emissions from industrial processes have fallen by 26% since 1990 to 343 million tonnes of CO<sub>2</sub>eq per year. These are emissions from processes that release greenhouse gases into the atmosphere other than through the combustion of fossil fuels - in the production of chemicals, cement, iron and steel, etc.  
```

Comment:
add comma
Emissions from industrial processes have fallen by 26% since 1990 to 343 million tonnes of 
to
Emissions from industrial processes have fallen by 26% since 1990, to 343 million tonnes of 

change
These are emissions from processes that release greenhouse gases into the atmosphere other than through the combustion of fossil fuels - in the production of chemicals, cement, iron and steel, etc.  
to
These emissions arise from processes that release greenhouse gases into the atmosphere other than through the combustion of fossil fuels - in the production of chemicals, cement, iron and steel, etc.

Reply:
Thanks, I updated the text:
1. Added comma after "1990"; and
2. Change the beginning of the line to start with: "These emissions arise"